db/migrate/20170907145915_create_salaries.rb

```diff
@@ -0,0 +1,12 @@
+class CreateSalaries < ActiveRecord::Migration[5.1]
+  def change
+    create_table :salaries do |t|
+      t.references :user, foreign_key: true
+      t.decimal :amount, precision: 8, scale: 2, default: 0.00
```

Comment:
I think we can just use `integer`. It is a really rare situation, when a guy can have a salary $1001.85